README.md

```diff
@@ -4,14 +4,35 @@ This operator is used to install, manage and configure an instance of
 [aws-load-balancer-controller](https://github.com/kubernetes-sigs/aws-load-balancer-controller/)
 in a OpenShift cluster.
 
+This [document](https://github.com/openshift/enhancements/blob/master/enhancements/ingress/aws-load-balancer-operator.md)
+describes the design and implementation of the operator in more detail.
+
+### Build the operand image
+
+The operand image must be built first. Our fork of the operand
+is [here](https://github.com/openshift/aws-load-balancer-controller/). Clone 
+the repository and build the image and push it to a registry which is 
+accessible from the test cluster.
+
+```bash
+git clone https://github.com/openshift/aws-load-balancer-controller.git
+IMG=quay.io/$USER/aws-load-balancer-controller
+podman build -t $IMG -f Dockerfile.openshift
+podman push $IMG
+```
+
 ### Running the operator
 
-1. Build and push the image to an image registry.
+1. Replace the operand image in the file `config/manager/manager.yaml` in 
+   the environment variable `RELATED_IMAGE_CONTROLLER` with the image 
+   created in the previous step.
+ 
+2. Build and push the image to an image registry.
     ```bash
-    export IMG=example.com/aws-load-balancer-operator:latest
+    export IMG=quay.io/$USER/aws-load-balancer-operator:latest
     make image-build image-push
     ```
-2. Create AWS credentials profile for the operator
+3. Create AWS credentials profile for the operator
```

Comment:
What I noticed trying these instructions is that the secret or credentials request is created in the operator namespace which doesn't exist yet as it'll be created in the last command: `make deploy`.